src/ParseServer.js

```diff
@@ -184,6 +184,19 @@ class ParseServer {
     api.use(middlewares.allowMethodOverride);
     api.use(middlewares.handleParseHeaders);
 
+    // add support of middlewares
+    if (options.middleware) {
```

Comment:
See contribution guide for how to add a new config option to Parse Server; the option needs to be validated, etc.

Reply:
hi.
* the comment is removed
* i do not add a new config option. the "middleware" option already exists